images/adswa/Singularity.nilearn--0.9.1

```diff
@@ -0,0 +1,117 @@
+# Generated by: Neurodocker version 0.7.0+0.gdc97516.dirty
+# Latest release: Neurodocker version 0.7.0
+# Timestamp: 2021/03/23 14:57:15 UTC
+# 
+# Thank you for using Neurodocker. If you discover any issues
+# or ways to improve this software, please submit an issue or
+# pull request on our GitHub repository:
+# 
+#     https://github.com/ReproNim/neurodocker
+
+Bootstrap: docker
+From: debian:stable-slim
+
+%post
+su - root
+
+export ND_ENTRYPOINT="/neurodocker/startup.sh"
+apt-get update -qq
+apt-get install -y -q --no-install-recommends \
+    apt-utils \
+    bzip2 \
+    ca-certificates \
+    curl \
+    locales \
+    unzip
+apt-get clean
+rm -rf /var/lib/apt/lists/*
+sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen
+dpkg-reconfigure --frontend=noninteractive locales
+update-locale LANG="en_US.UTF-8"
+chmod 777 /opt && chmod a+s /opt
+mkdir -p /neurodocker
+if [ ! -f "$ND_ENTRYPOINT" ]; then
+  echo '#!/usr/bin/env bash' >> "$ND_ENTRYPOINT"
+  echo 'set -e' >> "$ND_ENTRYPOINT"
+  echo 'export USER="${USER:=`whoami`}"' >> "$ND_ENTRYPOINT"
+  echo 'if [ -n "$1" ]; then "$@"; else /usr/bin/env bash; fi' >> "$ND_ENTRYPOINT";
+fi
+chmod -R 777 /neurodocker && chmod a+s /neurodocker
+
+apt-get update -qq
+apt-get install -y -q --no-install-recommends \
+    git
+apt-get clean
+rm -rf /var/lib/apt/lists/*
+
+export PATH="/opt/miniconda-latest/bin:$PATH"
+echo "Downloading Miniconda installer ..."
+conda_installer="/tmp/miniconda.sh"
+curl -fsSL --retry 5 -o "$conda_installer" https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash "$conda_installer" -b -p /opt/miniconda-latest
+rm -f "$conda_installer"
+conda update -yq -nbase conda
+conda config --system --prepend channels conda-forge
+conda config --system --set auto_update_conda false
+conda config --system --set show_channel_urls true
+sync && conda clean -y --all && sync
+conda create -y -q --name neuro
+bash -c "source activate neuro
+  pip install --no-cache-dir  \
+      "nilearn==0.9.1" \
+      "matplotlib""
+rm -rf ~/.cache/pip/*
+sync
+
+
+echo '{
+\n  "pkg_manager": "apt",
+\n  "instructions": [
+\n    [
+\n      "base",
+\n      "debian:stretch"
```

Comment:
it seems that you have some old recipe which you produced with neurodocker initially (for stretch, which is EOLed already) and then just modified here to use more recent debian etc, right?  

If so please either regenerate with fresh neurodocker or at least modify comment header on top of the file to reflect that it was modified and thus this metadata is no longer consistent with commands above.

Reply:
sorry for the late response - done! (added a comment)